src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/securityrule/AzureSecurityRulePlugin.java

```diff
@@ -82,7 +82,7 @@ public String requestSecurityRule(SecurityRule securityRule, Order majorOrder, A
     public void deleteSecurityRule(String securityRuleId, AzureUser azureUser) throws FogbowException {
         LOGGER.info(String.format(Messages.Info.DELETING_INSTANCE_S, securityRuleId));
 
-        SecurityRuleIdContext securityRuleIdContext = new SecurityRuleIdContext(securityRuleId);
+        SecurityRuleIdContext securityRuleIdContext = this.getSecurityRuleIdContext(securityRuleId);
```

Comment:
The changes made to this class should be in the azure-security-rules-plugin branch and not in this testing branch.